src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService_FindReferences.cs

```diff
@@ -48,6 +48,18 @@ internal abstract partial class AbstractFindUsagesService
             }
         }
 
+        async Task IFindUsagesLSPService.FindReferencesAsync(
+            Document document, int position, IFindUsagesContext context)
+        {
+            // We don't need to get third party definitions when finding references in LSP.
+            // Currently, 3rd party definitions = XAML definitions, and XAML will provide
+            // references via LSP instead of hooking into Roslyn.
+            // This also means that we don't need to be on the UI thread.
+            var definitionTrackingContext = new DefinitionTrackingContext(context);
+            await FindLiteralOrSymbolReferencesAsync(
+                document, position, definitionTrackingContext).ConfigureAwait(false);
+        }
```

Comment:
nit:

```suggestion
        Task IFindUsagesLSPService.FindReferencesAsync(Document document, int position, IFindUsagesContext context)
        {
            // We don't need to get third party definitions when finding references in LSP.
            // Currently, 3rd party definitions = XAML definitions, and XAML will provide
            // references via LSP instead of hooking into Roslyn.
            // This also means that we don't need to be on the UI thread.
            return FindLiteralOrSymbolReferencesAsync(document, position, new DefinitionTrackingContext(context));
        }
```